ansible/playbooks/ptf_installation.yaml

```diff
@@ -0,0 +1,81 @@
+---
+- name: PTF Installation
+  hosts: hana
+  remote_user: cloudadmin
+  become: true
+  become_user: root
+  vars:
+    ptf_url: "your_ptf_url"
+    ptf_user: "your_username_here"
+    ptf_password: "your_pass_here"
+    ptf_dir: "/tmp/ptf_dir"
+  tasks:
+    - name: Create directory for PTF installation
+      file:
+        path: "{{ ptf_dir }}"
+        state: directory
+        mode: '0755'
+
+    - name: Download PTF files recursively with wget
+      command: "wget --no-directories --recursive --reject 'index.html*' --user={{ ptf_user }} --password={{ ptf_password }} --no-parent {{ ptf_url }}"
+      args:
+        chdir: "{{ ptf_dir }}"
+
+    - name: List downloaded files
+      command: "ls -la"
+      args:
+        chdir: "{{ ptf_dir }}"
+      register: download_list
+
+    - name: Display downloaded files
+      debug:
+        var: download_list.stdout_lines
+
+    - name: Find downloaded RPM files
+      find:
+        paths: "{{ ptf_dir }}"
+        patterns: "*.rpm"
+      register: rpm_files
+
+    - name: Display found RPM files
+      debug:
+        var: rpm_files.files
+
+    - name: Filter out src.rpm files
+      set_fact:
+        filtered_rpm_files: "{{ rpm_files.files | selectattr('path', 'search', '^(?!.*src\\.rpm$).*') | list }}"
+
+    - name: Display filtered RPM files
+      debug:
+        var: filtered_rpm_files
+
+    - name: Install PTF RPM packages
+      zypper:
+        name: "{{ item.path }}"
+        state: present
+        disable_gpg_check: yes
+        update_cache: yes
+      loop: "{{ filtered_rpm_files }}"
+      loop_control:
+        label: "{{ item.path }}"
+
+    - name: Verify installation
+      command: "rpm -qf $(which fence_azure_arm)"
+      args:
+        _uses_shell: true
+      register: verify_installation
+
+    - name: Display verification result
+      debug:
+        var: verify_installation.stdout_lines
+
+    - name: Install additional Python packages
+      zypper:
+        name:
+          - python311-pexpect
+          - python311-pycurl
+        state: present
+        update_cache: yes
+
+    - name: Modify fence_azure_arm script to use Python 3.11
```

Comment:
all the tasks since here are generic. Could you move this last one in a different playbook please?